uiowa/uiowa
# Remove image background media type

Resolves #596 

## Problem
Image_background is available in SiteNow(P) (and as a side effect, collegiate sites as well)--used for section background images in paragraphs, but not used in layout builder.

## Resolution

- [x] Move existing image_background to standard image.
- [x] Remove image_background
- [x] Update paragraphs theming to apply proper style (ui_edgy_no_crop) using regular image for section backgrounds

## Testing
- Sync site. 
  - (If you haven't rsynced files, 'file does not exist' errors will be thrown. This is to be expected, and syncing or stage proxy after syncing should still work appropriately.)
- Check that image:background is no longer available as an option in media.
- If testing a SiteNow(P) site, check that any previous image:background media now appear as regular image media.
- Add a section background image using standard image media and check that it renders correctly, with the appropriate styling.
- Check that non-background images haven't changed, and are appropriately styled (dimensions appear normal, resolution appears normal)
